types/three/examples/jsm/nodes/core/AssignNode.d.ts

```diff
@@ -0,0 +1,17 @@
+import { ShaderNodeObject } from '../shadernode/ShaderNode.js';
+import Node from './Node.js';
+import TempNode from './TempNode.js';
+
+export default class AssignNode extends TempNode {
+    constructor(targetNode: Node, sourceNode: Node);
+
+    hasDependencies(): false;
+}
+
+export const assign: (targetNode: Node, sourceNode: Node) => ShaderNodeObject<AssignNode>;
```

Comment:
I think `nodeProxy` allows the arguments to be `NodeRepresentation`s that gets converted to `Node`s through the proxy implementation.

```suggestion
export const assign: (targetNode: NodeRepresentation, sourceNode: NodeRepresentation) => ShaderNodeObject<AssignNode>;
```

Reply:
Ah, that's how `nodeProxy` works. thank you!